tests/s3/test_object_tagging.py

```diff
@@ -759,7 +759,7 @@ def test_valid_specialchars_2473(self):
             self.object_name,
             self.file_path,
             mb_count=S3_OBJ_TST["s3_object"]["mb_count"],
-            key=S3_OBJ_TST["s3_object"]["key"],
+            key=S3_OBJ_TST["test_9429"]["key"],
```

Comment:
Is this related to changes from yaml?

Reply:
Yes, taking tag key as valid special characters.